src/pages/task/metadata/index.tsx

```diff
@@ -277,6 +278,12 @@ export const Metadata: React.VFC<Props> = ({
           ))}
         </span>
       ) : null}
+      {resetWhenFinished && (
```

Comment:
about placement: I'm not sure if this should fall below the `Depends On` section. The `Depends On` section could be very long. It might be better it we moved where this message displays